internal/security-governance-system/scanner-service/scanner-microService/README.md

```diff
@@ -0,0 +1,18 @@
+# Scanner Microservice API
+
+This is a Spring Boot microservice API which exposes the security scanner tool's APIs.
+
+This supports the following APIs.
+1. Start scan using the product zip file.
+2. Start the scan using the product's github location.
```

Comment:
There's just 1 start scan mechanism. No need of separate methods for product zip file and github location.